index.js

```diff
@@ -0,0 +1,2 @@
+require('angular-xml.js');
+module.exports = 'xml';
```

Comment:
Again you're right, but now it's what's expected. This way you can use it in the list of angular modules dependencies, for example:

``
var myApp = require('angular').module('myApp', [ require('angular-xml') ]);
``

It's a standard that's been adopted among major AngularJS modules (for [example angular-loading-bar](https://github.com/chieffancypants/angular-loading-bar/blob/master/index.js))